akka-bench-jmh/src/main/scala/akka/stream/FlowMapBenchmark.scala

```diff
@@ -62,7 +62,7 @@ class FlowMapBenchmark {
     val settings = ActorFlowMaterializerSettings(system)
```

Comment:
`ActorFlowMaterializerSettings` should be `ActorMaterializerSettings`, since now it is `ActorMaterializer`